src/styles/components/dropdown-panels/styles.less

```diff
@@ -0,0 +1,36 @@
+& when (@dropdown-panels-enabled) {
+
+  .dropdown-panel-group {
+    position: relative;
+
+    > .dropdown-panel {
+      box-shadow: @modal-shadow;
+      max-width: 100%;
+      position: absolute;
+      right: 0;
+      z-index: @z-index-overlay;
+
+      &.dropdown-panel-full-width {
+        width: 100%;
+      }
+
+      &.dropdown-panel-left {
+        left: 0;
+        right: auto;
+      }
+
+      &.dropdown-panel-animated {
+        margin-top: -10px;
+        opacity: 0;
+        pointer-events: none;
+        transition: margin-top @modal-animation-duration @modal-animation-easing, opacity @modal-animation-duration @modal-animation-easing;
```

Comment:
The complications with `transform` only occur when a direct descendent uses `position: fixed` as well. I can't see a reason for a child of this component to use `position: fixed`, and in the rare case that a child did need `position: fixed`, we would just use the `Portal` component to render at the root of `body`